x-pack/plugins/reporting/server/lib/content_stream.ts

```diff
@@ -6,9 +6,20 @@
  */
 
 import { Duplex } from 'stream';
+import { defaults, get } from 'lodash';
+import Puid from 'puid';
+import { ByteSizeValue } from '@kbn/config-schema';
 import type { ElasticsearchClient } from 'src/core/server';
 import { ReportingCore } from '..';
-import { ReportDocument } from '../../common/types';
+import { ReportSource } from '../../common/types';
+import { ExportTypesRegistry } from './export_types_registry';
+
+/**
+ * @note The Elasticsearch `http.max_content_length` is including the whole POST body.
+ * But the update/index request also contains JSON-serialized query parameters.
+ * 1Kb span should be enough for that.
```

Comment:
This should be fine, since `id`, `index`, `if_primary_term`, `if_seq_no` should all fit in that 1kb space.